prql-compiler/src/codegen.rs

```diff
@@ -295,8 +354,8 @@ fn binding_strength(expr: &pl::ExprKind) -> i32 {
             pl::BinOp::Or => 1,
             pl::BinOp::Coalesce => 2,
```

Comment:
Something is not right here. Looking at this https://github.com/prql/prql/blob/main/prql-compiler/src/parser/expr.rs#L191-L197, the precedence should be:

```
unary > range > mul,div,mod > add,sum > compare > coalesce > and > or
```

(I've tested that it does add some unnecessary parenthesis)